src/components/VmNics/index.js

```diff
@@ -39,15 +38,15 @@ class VmNic extends React.Component {
     const idPrefix = `vmnic-${nic.get('name')}`
     const nicInfoContent = (
       <div id={`${idPrefix}-nic-info`}>
-        {msg.network()}: {vnicProfile.getIn(['network', 'name'])}
+        {msg.network()}: {vnicProfile ? vnicProfile.getIn(['network', 'name']) : msg.noNetwork()}
         <br />
-        {msg.vnicProfile()}: {vnicProfile.get('name')}
+        {msg.vnicProfile()}: {vnicProfile ? vnicProfile.get('name') : msg.noNetwork()}
       </div>
     )
 
     const text = (
       <span className={style['light']} id={`${idPrefix}-nic`}>
-        ({vnicProfile ? `${vnicProfile.getIn(['network', 'name'])}/${vnicProfile.get('name')}` : null})
+        ({vnicProfile ? `${vnicProfile.getIn(['network', 'name'])}/${vnicProfile.get('name')}` : `${msg.noNetwork()}/${msg.noNetwork()}`})
```

Comment:
This will render:
`No network/No network` 
text to the user. I think simple
`No network`
is sufficient.
@jelkosz , what do you think?

Reply:
@mareklibra agreed